tests/acceptance/actions-test.js

```diff
@@ -48,6 +54,20 @@ test('allows to chain actions', function(assert) {
   });
 });
 
+test('allows to chain aliased actions', function(assert) {
```

Comment:
Aha...this makes complete sense. I feel like it would be most intuitive to for aliased methods to return the alias context. Given a page object that looks like this...
```javascript
const page = create({
  foo: {
  },

  bar: clickable('.btn1'),
  baz: clickable('.btn2'),

  clickFoo: alias('foo.click'),
  fillFoo: alias('foo.fillIn')
});
```
I think it would be unexpected if you could do `page.bar().baz()` but not `page.clickFoo().fillFoo('stuff')`. Does that seem like a reasonable point in favor of returning the alias context rather than the aliased action context?